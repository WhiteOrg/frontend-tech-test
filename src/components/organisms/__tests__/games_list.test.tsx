import React from "react";
import renderer from "react-test-renderer";
import GamesList from "../games_list";

describe("ORGANISMS: Render testing", () => {
  test("Should render GameList", () => {
    const data = {
      titleGamesList: "Include Games",
      games: [
        {
          title: "First Game",
          src: "https://i.imgur.com/gmXp0zM.jpg",
        },
        {
          title: "Second Game",
          src: "https://i.imgur.com/ecK4twD.jpg",
        },
        {
          title: "Third Game",
          src: "https://i.imgur.com/kn70JZB.jpg",
        },
        {
          title: "Fourth Game",
          src: "https://i.imgur.com/F6XZ6v5.jpg",
        },
        {
          title: "Fifth Game",
          src: "https://i.imgur.com/4RKCXxe.jpg",
        },
        {
          title: "Sixth Game",
          src: "https://i.imgur.com/lVq9s16.jpg",
        },
      ],
    };
    const component = renderer.create(<GamesList {...data} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
