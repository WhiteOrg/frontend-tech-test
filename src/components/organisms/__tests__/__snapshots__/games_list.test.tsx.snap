// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ORGANISMS: Render testing Should render GameList 1`] = `
<section
  className="sc-AxiKw jWoTjM"
>
  <h2
    className="sc-AxirZ hvUUMv"
    color="#091870"
  >
    Include Games
  </h2>
  <div
    className="sc-AxhCb ckJXts"
  >
    <img
      alt="First Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/gmXp0zM.jpg"
    />
    <img
      alt="Second Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/ecK4twD.jpg"
    />
    <img
      alt="Third Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/kn70JZB.jpg"
    />
    <img
      alt="Fourth Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/F6XZ6v5.jpg"
    />
    <img
      alt="Fifth Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/4RKCXxe.jpg"
    />
    <img
      alt="Sixth Game"
      className="sc-AxhUy gVYkPg"
      src="https://i.imgur.com/lVq9s16.jpg"
    />
  </div>
</section>
`;
